src/osg/Texture3D.cpp

```diff
@@ -315,30 +315,20 @@ void Texture3D::apply(State& state) const
     }
     else if ( (_textureWidth!=0) && (_textureHeight!=0) && (_textureDepth!=0) && (_internalFormat!=0) )
     {
-        // no image present, but dimensions at set so lets create the texture
```

Comment:
think this whole file can go back to the 3.6 version now that selectSizedInternalFormat is going back into Texture.cpp